### PR TITLE
Code from trials

### PR DESCRIPTION
**Pull request by krisbobo**
## Project: Review Project from Trials ##
In the project, I did the following:
- Copied code from my past trials project to learn the code review process.

### CODE REVIEW

**Review by krisbobo — COMMENTED** [review]
# Highlights #
Hi! You did a great job with the following:
- Descriptive PR
- Good job documenting the README file in a professional manner.
You did a good job implementing some or all of the project requirements. The following are required:
## REQUIRED CHANGES ##
- Ensure all linter checks pass.
- Check the sub-comments for further changes required.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+      <header>Reference</header>
+      <article>
+        <ul>
+          <li>
+            All the documentation in this page is taken from
+            <a
+              href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide"
+              target="_blank"
+              >MDN</a
+            >
```
Comment (krisbobo) [inline index.html:432]: I suggest you place all closing tags on the same line to ease the readability of your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+      <li><a class="nav-link" href="#Data_types">Data types</a></li>
+      <li>
+        <a class="nav-link" href="#if...else_statement">if...else statement</a>
+      </li>
+      <li><a class="nav-link" href="#while_statement">while statement</a></li>
+      <li>
+        <a class="nav-link" href="#Function_declarations"
+          >Function declarations</a
+        >
+      </li>
```
[the remark this answers is not in the document]
Comment (danmainah) [reply index.html:37]: - [ ]  @krisbobo Great review, I loved how you explained why the changes need to be made. I would suggest highlighting the sections specifically where the changes need to be made, like in this case line 34 to 36, that way its even more easier to see the problem for the deveoper.

**Review by danmainah — APPROVED** [review]
Hi @krisbobo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good design of the review summary.
✔️  Good use of inline comments.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**